DjangoProject/authenticationapp/forms.py

```diff
@@ -0,0 +1,24 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
```

Comment:
Please follow PEP8 guidelines for all the new imports used in this PR.

> Imports should be grouped in the following order:
> 
>     standard library imports
>     related third-party imports
>     local application/library specific imports
> 
> You should put a blank line between each group of imports.

In Two scoops of Django, you can find it at the beginning.
[https://www.python.org/dev/peps/pep-0008/#imports](url)